proto/regen/ecocredit/v1alpha2/types.proto

```diff
@@ -30,18 +30,32 @@ message ClassInfo {
   uint64 num_batches = 6;
 }
 
+// ProjectInfo represents the high-level on-chain information for a project.
+message ProjectInfo {
+
+  // project_id is the unique ID of the project.
+  string project_id = 1;
```

Comment:
This is a good point @clevinson. I agree it seems important we include information about the 'origin registry'. I think this will help reduce complexity and makes it a lot easier to find the original issuer of the credits. I think both solutions would work well. If we choose your second solution I think it would make more sense to put the `project_registry` field in  `ProjectInfo` rather than the credit class because it's still a bit unclear how the credit class will/can be used. For instance, someone might define a credit class to allow methodologies from multiple registries. Storing it in `ProjectInfo` makes more sense because the project itself should only be registered with a single registry.